lms/djangoapps/verify_student/tests/test_views.py

```diff
@@ -1776,22 +1784,20 @@ def test_incourse_reverify_get(self):
         self.mock_tracker.reset_mock()
 
     @patch.dict(settings.FEATURES, {'AUTOMATIC_VERIFY_STUDENT_IDENTITY_FOR_TESTING': True})
-    @patch('verify_student.views.render_to_response', render_mock)
-    def test_invalid_checkpoint_post(self):
-
-        response = self.client.post(self._get_url(self.course_key, self.MIDTERM))
-        self.assertEquals(response.status_code, 200)
-        ((template, context), _kwargs) = render_mock.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertIn('incourse_reverify', template)
-        self.assertTrue(context['error'])
+    def test_checkpoint_post(self):
+        """
+        Test that sending user gets 400 (Bad Request) response on sending
+        POST request with invalid checkpoint location.
```

Comment:
Could you clarify this? Maybe `Verify that POST requests including an invalid checkpoint location result in a 400 response`.